src/platform/nRF5/BLEManagerImpl.cpp

```diff
@@ -88,6 +106,57 @@ BLEManagerImpl BLEManagerImpl::sInstance;
 #define CHIP_DEVICE_LAYER_BLE_OBSERVER_PRIORITY 3
 #endif // CHIP_DEVICE_LAYER_BLE_OBSERVER_PRIORITY
 
+#if defined(SOFTDEVICE_PRESENT)
+
+static void OnSoCEvent(uint32_t sys_evt, void * p_context)
+{
+#if CHIP_ENABLE_OPENTHREAD
+    otSysSoftdeviceSocEvtHandler(sys_evt);
+#endif
+    UNUSED_PARAMETER(p_context);
+}
+
+#endif // defined(SOFTDEVICE_PRESENT)
+
+static uint32_t LogTimestamp(void)
+{
+    return 0;
+}
+
+CHIP_ERROR BLEManagerImpl::_PlatformInit()
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+#if defined(SOFTDEVICE_PRESENT)
+
+    err = nrf_sdh_enable_request();
```

Comment:
Sorry, I have to ask about this too. Why is this being initialized in **both** the lock/lighting app main.cpp and in the BLEManager?